### DISCUSSION TITLE
Why Dist directory not generated anymore?

### DISCUSSION THREAD

**Discussion by gaelgerard** [holger1411/understrap#1429] [Q&A] [answered]
Bonjour,

I use a custom Understrap child theme for several years now With a forked Understrap 0.94 version.

As I am beginning a long New project I decided to begin with a fresh new install with latest versions of both parent and child Understrap themes.

I was surprised to see Gulp disappearing but more annoying for my workflow is the dist directory which is not generated anymore?

Why does `npm run dist` not generate a dist directory with only necessary files for my theme to work?

Merci de votre aide !

**Comment by bacoords** [comment, marked as answer]
Thanks for the question. 

In our discussions here, we heard that that functionality wasn't highly used and that developers were distributing their child themes while keeping their sass and other source files in the theme folder for future development. 

However, if that's not the case and developers like being able the `/dist/` folder, we can reintroduce it as a new command. For now, you can always pull an earlier version of the child theme from the releases page and continue using that: https://github.com/understrap/understrap-child/releases/tag/v0.6.0

**Reply by iwasthesword** [reply]
The explanation doesn't make too much sense to me because even if you use the child theme you still need to `npm install` and all the modules will be inside the child theme and you need to clean before uploading. 
I'm also for bringing `npm run dist` command back as it was part of my workflow using the child theme.

**Reply by bacoords** [reply]
I can see your point. I guess it depends on how you're distributing your child theme (via git, sftp, some other method). 

I've added a new issue so we can look into this:  https://github.com/understrap/understrap/issues/1431

**Reply by bacoords** [reply]
I've added a PR to cover this issue: https://github.com/understrap/understrap/pull/1432

 It's a pretty simple change if you don't want to wait for the next release. Just check out the PR and:
- add the two new files to `/src/build`
- add the two new commands to the `package.json` file.
